third_party/move/tools/move-coverage/src/source_coverage.rs

```diff
@@ -42,6 +46,98 @@ pub enum AbstractSegment {
     BoundedLeft { start: u32 },
 }
 
+#[derive(ValueEnum, Clone, Debug, Serialize)]
+pub enum ColorChoice {
+    None,
+    Default,
+    Always,
+}
+
+impl Display for ColorChoice {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        use ColorChoice::*;
+        match self {
+            None => f.write_str("none"),
+            Default => f.write_str("default"),
+            Always => f.write_str("always"),
+        }
+    }
+}
+
+impl FromStr for ColorChoice {
+    type Err = &'static str;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        use ColorChoice::*;
+        match s {
+            "none" => Ok(None),
+            "default" => Ok(Default),
+            "always" => Ok(Always),
+            _ => Err("unknown variant"),
+        }
+    }
+}
+
+#[derive(ValueEnum, Clone, Debug, Serialize)]
+pub enum TextIndicator {
```

Comment:
Would be good to have a brief note on what these mean.

Reply:
Done.